### DISCUSSION TITLE
IMSAI Front Panel Updated Cheat Sheet - October 20, 2023 Version - Unofficial Documentation

### DISCUSSION THREAD

**Discussion by nbreeden2** [thehighnibble/thehighnibble.github.io#116] [IMSAI8080esp replica]
I've updated the Front Panel Cheat Sheet to update it and make corrections.
As I seem to have lost the original spreadsheet used to generate the original PDF I've had to recreate it, it's included in the ZIP file.
As this is a new release (October 20, 2023) there may be typos or errors I've missed. Please let me know.
This includes updated notes, features and updated examples. It should be current with the current release of the emulator. 
Hoping someone can verify the file is publicly available on the Dropbox link below. 

Edited to update the link to the latest version:
https://www.dropbox.com/scl/fi/l4jvfo0x9dlx8tmjvman1/Switch-Setting-Cheat-Sheet-10202023-2.zip?rlkey=ou6runuhzwmu9xblyf0vxbqz1&dl=0

**Comment by laffineuse** [comment]
Thanks for this!  very clear
